rdmo/projects/templates/projects/project_interview_contact_help.html

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
+
+    <p>
+        {% blocktrans trimmed %}
+        If you have questions about a question or other inqueries to support, you can use the
```

Comment:
maybe:
```suggestion
If you have a question or feedback please get in touch with us, you can use this form to send an email to the {{ request.site.name }} support"

```
or is it meant to be overwritten in the rdmo-app anyway?

Reply:
I changed it, and yes it is supposed to be overwritten, but a good default does not hurt.